src/styles/card.css

```
@@ -0,0 +1,88 @@
+
+.card {
+  display: flex;
+  flex-direction: column;
+  background: var(--white-color);
+  border: 1px solid var(--white-color);
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  max-width: 200px;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+  margin-bottom: 40px;
+  align-self: center;
```

do you need it?

---

you can do it with 1 style
```suggestion
    margin: 0 auto 40px;
```
or remove flex from card & add width: 100% here